src/views/Conference.vue

```diff
@@ -1,36 +1,59 @@
 <template>
   <Page class="league-page-container">
+    <Loading v-show="loading" />
```

Comment:
I would suggest using `v-if` by default as far as it works simpler than `v-show`
```suggestion
    <Loading v-if="loading" />
```

Reply:
I can't use `v-if` because jitsi instance needs some container to make sure that iframe if loaded.

Reply:
Yes, I got it but seems that the Loader doesn't need `v-show`

Reply:
True